tests/test-utils.js

```diff
@@ -0,0 +1,81 @@
+import path from 'path';
```

Comment:
Maybe we should leave this out for now until there is consensus about the initial test setup